Library/Formula/teensy_loader_cli.rb

```diff
@@ -12,9 +12,18 @@ class TeensyLoaderCli < Formula
     sha256 "dcd10140babb4d2937ce376c89e9c24a2e8046d2cabdad2cfdbc2542afa14471" => :mavericks
   end
 
+  head do
+    url "https://github.com/PaulStoffregen/teensy_loader_cli.git"
+    patch do
```

Comment:
Please submit this patch upstream (or at least a bug report) and use that link. We don't particularly want to use a commit from an unofficial repo.